index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    
+    <header class="nav-bar">
+           <a class="logo" href="#">My logo</a>
+           <a class="menu-button" href="#"><img src="/images/Menu.png" alt=""></a>
+
+    </header>
+
+    <section class="headline">
+      <h2> I’m Emirjeta <br> Glad to see <br>you!</h2>
```

Comment:
- [ ] [REQUIRED] Your main title is currently an h2 tag. please change this main heading of the page to the h1 tag. This is a part of good SEO.